libs/domain-core/src/lib/aggregate-repository.spec.ts

```diff
@@ -57,15 +57,17 @@ describe('AggregateRepository', () => {
       expect(testEventStore.read).toHaveBeenCalledTimes(1);
       expect(testEventStore.read).toHaveBeenNthCalledWith(1, stream);
 
-      expect(testAggregate.applyEvent).toHaveBeenCalledTimes(1);
-      expect(testAggregate.applyEvent).toHaveBeenNthCalledWith(1, testEvent);
+      expect(testAggregate.applyEvent).toHaveBeenCalledTimes(2);
+      expect(testAggregate.applyEvent).toHaveBeenNthCalledWith(1, testEventFirst);
+      expect(testAggregate.applyEvent).toHaveBeenNthCalledWith(2, testEventSecond);
       expect(aggregate.revision).toBeGreaterThan(-1);
```

Comment:
This accurately tests the version. What say?

```suggestion
      expect(aggregate.revision).toBeGreaterThan(1);
```